packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -47,6 +47,12 @@ export interface IButtonProps extends React.HTMLAttributes<HTMLButtonElement | H
   checked?: boolean;
 
   /**
+   * When true, and an icon is passed in, the icon will be hidden until button is checked.
+   * @default false
+   */
+  canCheck?: boolean;
```

Comment:
I think we should have an icon which is used for checked vs unchecked and the checked prop should drive that. If the creator doesn't want to represent checked state with an icon they can not use that prop. Thoughts?